File: python/day02.py
def patternPrint(spaces,asterics):
    for i in range (0,spaces):
        print(" ",end="")
    for i in range(0,asterics):
        print("*",end="")
    print("\n")


n = int(input("enter height of pattern"))
counter=2
for i in range(0,n):
    patternPrint(n-i,counter)
    patternPrint(n-i,counter)
    counter = counter + 2

File: cpp/day02.cpp
/*
Print this pattern.
            **
            **
           ****
           ****
          ******
          ******
         ********
         ********
*/
#include<bits/stdc++.h>

using namespace std;

void patternPrint(int spaces,int asterics){
    for(int i=0;i<spaces;i++)
        printf(" ");
    for(int i=0;i<asterics;i++)
        printf("*");
    printf("\n");

}
int main(){
    int n;
    cin>>n;
    int counter=2;
    for(int i=0;i<n;i++){
        patternPrint(n-i,counter);
        patternPrint(n-i,counter);
        counter+=2;
    }

}
